tests/conftest.py

```diff
@@ -876,3 +877,20 @@ def towns_fund_round_6_file_failure() -> Generator[BinaryIO, None, None]:
         "rb",
     ) as file:
         yield file
+
+
+@pytest.fixture
+def setup_organisationadmin(test_session):
+    view = OrganisationAdminView(test_session)
+    view.fields_allowed_to_edit = ["external_reference_code"]
```

Comment:
Why do we need to instantiate this here? This feels quite unusual 🤔

Reply:
Yes, it was totally unnecessary.